src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/RepresentedOrganisationDetails.cshtml

```diff
@@ -14,12 +14,60 @@
         @Html.Partial("ViewOrganisation/_OrganisationDetailsTabs", Model)
 
         <div class="govuk-tabs__panel" id="organisationDetails">
```

Comment:
@stevelee-civica  do we need to conditionally show this when its an authed rep or is it handled elsewhere?

Reply:
inside _OrganisationDetailsTabs.cshtml